doc/classes/PhysicsDirectBodyState2D.xml

```diff
@@ -139,6 +139,15 @@
 				[b]Note:[/b] By default, this returns 0 unless bodies are configured to monitor contacts. See [member RigidBody2D.contact_monitor].
 			</description>
 		</method>
+		<method name="get_contact_impulse" qualifiers="const">
```

Comment:
Use tabs for indentation, *except* for code samples. Right now, there is mixed tab and space indentation.

Since this method doesn't have code samples, all indentation should use tabs.